db/migrate/20230719004620_add_fields_to_hackathons.rb

```diff
@@ -0,0 +1,9 @@
+class AddFieldsToHackathons < ActiveRecord::Migration[7.0]
+  def change
+    add_column :hackathons, :website, :string, null: false
+    add_column :hackathons, :high_school_led, :boolean, null: false
```

Comment:
If being high school led is a requirement, why are we adding a bool for it if it's always true?

Reply:
The review flow for Ops is that they'll still look at Hackathons that aren't high school led, then reach out to them to say that they aren't approved because being high school led is a requirement.

I'm not exactly sure why the review process is like this, however, it does appear that Hack Club loves collecting form entries even if that entry will be easily rejected. Some examples include this hackathons form and also the form at https://hackclub.com/slack

![image](https://github.com/hackclub/hackathons-backend/assets/20099646/cfc89418-9deb-4f48-bd84-0cfe0b97a00f)
![image](https://github.com/hackclub/hackathons-backend/assets/20099646/dbdca13a-0d3f-4bc6-9a58-b2e1cb26f5dc)

Reply:
I was gonna add a `has_rich_text :description` anyway, so applicants would be able to say what their hackathon is about, and they'll have to include a website anyway, so we don't have to add a redundant column when we can just include the requirement on the application.

Reply:
[`7e68e43` (#53)](https://github.com/hackclub/hackathons-backend/pull/53/commits/7e68e437e2ac6b97b9641cd759c0f267f8881c5d)